mobile/src/stateMachine/views/main/home.tsx

```diff
@@ -166,11 +116,11 @@ export class HomeView extends ViewState<{ opacity: Animated.Value, isUnfinishedQ
     }
 
     // Not used currently
-    private async onStartDomains() {
-        AppViewModel.Instance.Domain.clearSelectedDomains();
-        AppViewModel.Instance.Strategy.clearSelectedDomains();
-        this.trigger(ScenarioTriggers.Senary);
-    }
+    // private async onStartDomains() {
+    //     AppViewModel.Instance.Domain.clearSelectedDomains();
+    //     AppViewModel.Instance.Strategy.clearSelectedDomains();
+    //     this.trigger(ScenarioTriggers.Senary);
+    // }
```

Comment:
Why is this commented out? If needed, please leave a code comment explaining why it's commented out.